### ISSUE TITLE
No posible more than one instance of brakets in Windows8.1

### ISSUE THREAD

**Issue by emaza** [adobe/brackets#11503]
In previous versions to brakets 1.4 I can have more than one braquets open at time.
I update to Realease 1.4 and now I can´t open an other brakets program at same time.
I use Windows 8.1
Thanks

**Comment by robert52** [comment]
I can confirm that too.

**Comment by petetnt** [comment]
I think that for a long time the goal has been to run Brackets as an single instance and using `Debug -> New Brackets Window` to open a new actual Brackets-window. However, it has been possible to launch a new instance of Brackets by, for example, right clicking the icon like I showcased on #10769. 

I do think that right clicking the icon should have a `New Brackets Window` option for ease of use as it's kinda hard to find under Debug and it's often useful to have to different windows running.

**Comment by robert52** [comment]
@petetnt I really like the idea. Google Chrome and many more have a `Tasks` section when you right-click on the icon, and various tasks can be executed from there.

**Comment by NathanSTG** [comment]
I experience this on windows 7 x64 after update as well. I'm fine with the idea of a single instance, but I agree it would be very helpful to have the "New Window" option listed in the taskbar icon context menu.

**Comment by mmkaw** [comment]
I Agree. any one having problem when opening new window form menu. CTRL does not work on the second window, like type anything and hit ctrl+z which should undo but doesn't actually. opened the issue but no response from anyone.
https://github.com/adobe/brackets/issues/11531

**Comment by NathanSTG** [comment]
I am not experiencing the problem with CTRL not working on the second window. Everything seems to work fine in both windows.

**Comment by mmkaw** [comment]
Yeah its working fine now, may be problem was with my system. have already closed the issue. Thank you.

**Comment by firaskudsy** [comment]
i also face the same issue with the CTRL. before  update i was able to open multiple projects windows, now only one !!
